common/compositor/va/varenderer.h

```diff
@@ -17,13 +17,37 @@
 #ifndef COMMON_COMPOSITOR_VA_VARENDERER_H_
 #define COMMON_COMPOSITOR_VA_VARENDERER_H_
 
+#include <map>
 #include "renderer.h"
+#include "va/va.h"
+#include "va/va_vpp.h"
 
 namespace hwcomposer {
 
 struct OverlayLayer;
 class NativeSurface;
 
+typedef enum _VppColorBalanceMode {
+  COLORBALANCE_NONE = 0,
+  COLORBALANCE_HUE,
+  COLORBALANCE_SATURATION,
+  COLORBALANCE_BRIGHTNESS,
+  COLORBALANCE_CONTRAST,
+  COLORBALANCE_CONUNT,
+} VppColorBalanceMode;
+
+typedef struct _VppColorBalanceCap {
+  float min;  //query from va
```

Comment:
You can use the structure like this, it will save one by one assign at L241. This just a recommendation, you can choose change or not

struct VppColorBalanceCap {
   VAProcFilterCapColorBalance caps; //query from qa
    float value; 
};

Reply:
@xuguangxin , I have fixed the issue you mentioned, Please have a review. Thanks.